src/pages/kb/user-guide/querying/query-parameters.md

```diff
@@ -124,6 +124,20 @@ WHERE org_id = {{org_id}} AND created_at > '{{start_date}}'
 
 We use two parameters: `{{org_id}}` and `{{start_date}}`.
 
+**Can I use parameters in embedded visualizations?**
+
+Yes, with one exception. If a query uses a Text type parameter, it cannot be embedded because text parameters are not safe from SQL injection. All other types of query parameters can be safely embedded in visualizations.
```

Comment:
Maybe worth pointing out that dropdowns with text are fine, it's just the explicit Text parameter type that is of an issue.

Reply:
Also worth mentioning that this is true from Redash v8 or the hosted one.

Reply:
> Maybe worth pointing out that dropdowns with text are fine, it's just the explicit Text parameter type that is of an issue.

I feel that @susodapop's phrasing is pretty explicit.

Reply:
And here's why I thought it's worth making it explicit:

![image](https://user-images.githubusercontent.com/71468/61619123-ec2cfb00-ac76-11e9-8a91-4a02c3dc6853.png)

I would consider going as far as making a table with all parameter types and a column marking if they are safe or not.

@susodapop @rauchy @ranbena

Reply:
I'll adjust this and open a fresh PR